src/main/java/frc/robot/subsystems/flywheel/factory/FlywheelFactory.java

```diff
@@ -3,30 +3,30 @@
 import frc.robot.Robot;
 import frc.robot.hardware.motor.sparkmax.SparkMaxDeviceID;
 import frc.robot.subsystems.flywheel.Flywheel;
-import frc.robot.subsystems.flywheel.FlywheelComponents;
+import frc.robot.subsystems.flywheel.FlywheelStuff;
 import frc.robot.subsystems.flywheel.FlywheelConstants;
 
 public class FlywheelFactory {
 
 	//@formatter:off
-	private static Flywheel generateReelFlywheel(
-			String topLogPath,
-			String bottomLogPath,
-			SparkMaxDeviceID topID,
-			SparkMaxDeviceID bottomID,
-			Robot robot
+	private static Flywheel generateRealFlywheel(
+		String topLogPath,
+		String bottomLogPath,
+		SparkMaxDeviceID topID,
+		SparkMaxDeviceID bottomID,
+		Robot robot
 	) {
-		FlywheelComponents topComponents = RealFlywheelConstants
-			.generateFlywheelComponent(topLogPath, FlywheelConstants.IS_TOP_MOTOR_INVERTED, topID);
-		FlywheelComponents bottomComponents = RealFlywheelConstants
-			.generateFlywheelComponent(bottomLogPath, FlywheelConstants.IS_BOTTOM_MOTOR_INVERTED, bottomID);
+		FlywheelStuff topStuff = RealFlywheelConstants
+				.generateFlywheelStuff(topLogPath, FlywheelConstants.IS_TOP_MOTOR_INVERTED, topID);
+		FlywheelStuff bottomStuff = RealFlywheelConstants
+				.generateFlywheelStuff(bottomLogPath, FlywheelConstants.IS_BOTTOM_MOTOR_INVERTED, bottomID);
```

Comment:
use one tab, not two
```suggestion
		FlywheelStuff topStuff = RealFlywheelConstants
			.generateFlywheelStuff(topLogPath, FlywheelConstants.IS_TOP_MOTOR_INVERTED, topID);
		FlywheelStuff bottomStuff = RealFlywheelConstants
			.generateFlywheelStuff(bottomLogPath, FlywheelConstants.IS_BOTTOM_MOTOR_INVERTED, bottomID);
```